### PR TITLE
Feature/mds 2189

### PR DESCRIPTION
**Pull request by mmumby**
Creating the initial View for NoW applications
- There is no logic in this pr, only style and structure of the UI
- Will add unit tests in a follow up PR once logic is added
- This is also only visible in development

### CODE REVIEW

File: frontend/src/components/noticeOfWork/NOWActivities.js [hunk 1]
```diff
@@ -0,0 +1,1060 @@
...
+const recColumns = [
+  {
+    title: "Activity",
+    dataIndex: "activity",
+    key: "activity",
+  },
+  {
+    title: "Total Effected Area (ha)",
+    dataIndex: "efectedArea",
+    key: "efectedArea",
```
Comment (NWCalvank) [inline frontend/src/components/noticeOfWork/NOWActivities.js]: Typo: `effectedArea`

File: frontend/src/components/noticeOfWork/NOWActivities.js [hunk 2]
```diff
@@ -0,0 +1,1060 @@
...
+            <Panel header="Surface Bulk Sample" key="10">
+              {this.renderSurfaceBulkSample()}
+            </Panel>
+            <Panel header="Underground Exploration" key="11">
+              {this.renderUnderGround()}
+            </Panel>
+            <Panel header="Water Supply" key="12">
+              {this.renderWaterSupply()}
+            </Panel>
+          </Collapse>
```
Comment (NWCalvank) [inline frontend/src/components/noticeOfWork/NOWActivities.js]: Big fan of how this is organized

File: frontend/src/components/noticeOfWork/NoticeOfWorkApplication.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+import NOWGeneralInfo from "@/components/noticeOfWork/NOWGeneralInfo";
+import NOWWorkPlan from "@/components/noticeOfWork/NOWWorkPlan";
+import { AuthorizationGuard } from "@/HOC/AuthorizationGuard";
+/**
+ * @class NoticeOfWorkApplication - contains all information regarding to a notice of work application
+ */
+
+const { Panel } = Collapse;
+
+// eslint-disable-next-line react/prefer-stateless-function
```
Comment (NWCalvank) [inline frontend/src/components/noticeOfWork/NoticeOfWorkApplication.js]: What's the reason for needing this?

File: frontend/src/constants/routes.js [hunk 4]
```diff
@@ -159,3 +160,19 @@ export const SEARCH_RESULTS = {
...
+
+// export const NOTICE_OF_WORK_APPLICATIONS = {
+//   route: "/dashboard/notice-of-work",
+//   dynamicRoute: ({ page, per_page, ...params }) =>
+//     `/dashboard/notice-of-work?${queryString.stringify(
+//       { page, per_page, ...params },
+//       { sort: false }
+//     )}`,
+//   component: COMPONENT_NAME_GOES_HERE,
+// };
```
Comment (NWCalvank) [inline frontend/src/constants/routes.js]: Remove? If not, perhaps you could add a TODO to explain why it's here but commented out

Comment (mmumby) [reply frontend/src/constants/routes.js]: ah, yes forgot about this, removing

File: frontend/src/components/noticeOfWork/NoticeOfWorkApplication.js (hunk 3 above)
Comment (mmumby) [reply frontend/src/components/noticeOfWork/NoticeOfWorkApplication.js]: This component will be stateful once I start working with the data

Comment (NWCalvank) [reply frontend/src/components/noticeOfWork/NoticeOfWorkApplication.js]: Gotcha 👍

**Pull request merged**